helper/geojsonify_meta_data.js

```diff
@@ -8,7 +8,14 @@ var Document = require('pelias-model').Document;
  */
 function addMetaData(src, dst) {
   dst.id = src._id;
-  dst.gid = makeGid(src);
+
+  // at this time the gid's of interpolated addresses are not usable in the place endpoint
+  // to avoid confusion they will be removed altogether for now and replaced in the future when we have determined
+  // the right format for these types of results
+  if (src.source !== 'mixed') {
```

Comment:
the `source` returned by the interpolation service can be one of `mixed, openstreetmap, openaddresses`.

it is currently possible, due to builds being performed at different times, that an ID returned via the interpolation service (for OSM or OA) is not retrievable via the `/v1/place` endpoint.

see: https://github.com/pelias/api/blob/master/middleware/interpolate.js#L108-L111

I suspect this is more of an exception than the norm (more common for missing unit numbers than missing house numbers), if you think it might be a problem then we can consider removing the `gid` for all results coming back from the interpolation service by targeting `hit.match_type = 'interpolated'`.

see: https://github.com/pelias/api/blob/master/middleware/interpolate.js#L102